xonsh/built_ins.py

```diff
@@ -528,17 +531,21 @@ def run_subproc(cmds, captured=True):
                        universal_newlines=uninew)
         else:
             prev_is_proxy = False
+            
+            usetee = (stdout is None) and (not background) and \
+                     ENV.get('XONSH_STORE_STDOUT', False) 
+            cls = TeePTYProc if usetee else Popen
             subproc_kwargs = {}
-            if ON_POSIX:
+            if ON_POSIX and cls is Popen:
```

Comment:
I haven't tested this yet, but I am worried that without the `_subproc_pre`, job control will probably not work as expected...anyway, will test.